internal/powerplatform/services/licensing/resource_billing_policy.go

```diff
@@ -169,6 +175,17 @@ func (r *BillingPolicyResource) Create(ctx context.Context, req resource.CreateR
 		billingPolicyToCreate.Status = plan.Status.ValueString()
 	}
 
+	// Create() is passed a default timeout to use if no value
+	// has been supplied in the Terraform configuration.
+	createTimeout, diags := plan.Timeouts.Create(ctx, 20*time.Minute)
```

Comment:
All these hard-coded time durations would probably be better managed in a variable somewhere.

Reply:
This is a default and can be overridden by a variable/property on the resource.